.circleci/config.yml

```diff
@@ -43,7 +44,7 @@ commands:
       - checkout
       - run:
           name: Generate cache key
-          command: find . -type f -name "*.gradle*" -exec shasum {} + | sort > /tmp/checksum.txt && cat /tmp/checksum.txt
+          command: find . -type f -name "*.gradle-wrapper*" -exec shasum {} + | sort > /tmp/checksum.txt && cat /tmp/checksum.txt
```

Comment:
This change will cause cache misses, no files will match this pattern and the cache will never be updated. If it's being propagated from another place, that should be fixed. The intent is to key off any changes in the gradle code or the gradle (wrapper) version.

 The most accurate version of this command is probably
```bash
find . -type f -name "*.gradle*" -o -name "gradle-wrapper*" | xargs shasum | sort > /tmp/checksum.txt && cat /tmp/checksum.txt
```
Which produces the below on a test repo
```
33b0acb4572934fc1dd7d5880c65036724974e06  ./gradle/wrapper/gradle-wrapper.jar
6e2b9651ce90b9a8de1304134e4c0ef81ecfae80  ./service-impl/build.gradle.kts
723f2062a71251313edc9e6c3cadce31da8f89ed  ./settings.gradle.kts
96abe7707ce331aa7b3cf1961fe8ebe9e812c679  ./gradle/wrapper/gradle-wrapper.properties
a9e12dd6e753e9ba5a360ce1cf79d3afcaeacd94  ./service/build.gradle.kts
aed929c4fb91822a2ed57268c91dd60cafbf4c64  ./build.gradle.kts
bf3a8361aea58099052e4b2a4db0968da70568d0  ./service-api/build.gradle.kts
e47aaf32c16b04267fd91d0c183f9526eb62ab0c  ./semantic-build-versioning.gradle
```

Reply:
got it.

Reply:
will check if this is the case somewhere else as well.